lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -3170,19 +3169,32 @@ def model_add_material(model, material_name)
       end
     end
 
-    # OpenStudio::logFree(OpenStudio::Info, 'openstudio.standards.Model', "Adding material: #{material_name}")
-
     # Get the object data
-    data = model_find_object(standards_data['materials'], 'name' => material_name)
-    unless data
-      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.Model', "Cannot find data for material: #{material_name}, will not be created.")
-      return false
-      # @todo change to return empty optional material
+    if material_name.include?('Simple Glazing')
```

Comment:
I suggest checking for the data first, and then assign by name, instead of leading with the construction name.